mtgjson5/providers/tcgplayer.py

```diff
@@ -100,6 +106,13 @@ class TCGPlayerProvider(AbstractProvider):
         "3x Magic Booster Packs",
         "Booster Battle Pack",
     ]
+    booster_box_default_size = {
```

Comment:
not sure if booster_box is the correct term anymore

Reply:
it's not, moving to product_default_size, but for now not converting it to a dict